components/match2/wikis/valorant/match_group_input_custom.lua

```diff
@@ -76,28 +78,175 @@ end
 --
 --
 -- function to sort out winner/placements
-function CustomMatchGroupInput._placementSortFunction(table, key1, key2)
-	local op1 = table[key1]
-	local op2 = table[key2]
-	local op1norm = op1.status == 'S'
-	local op2norm = op2.status == 'S'
-	if op1norm then
-		if op2norm then
-			return tonumber(op1.score) > tonumber(op2.score)
-		else return true end
-	else
-		if op2norm then return false
-		elseif op1.status == 'W' then return true
-		elseif op1.status == 'DQ' then return false
-		elseif op2.status == 'W' then return false
-		elseif op2.status == 'DQ' then return true
-		else return true end
+-- function to check for draws
+function CustomMatchGroupInput.placementCheckDraw(table)
```

Comment:
```suggestion
function CustomMatchGroupInput.isPlacementADraw(table)
```

Do the same with the other bool functions. I would also add: all of these functions use a namespace inside the name ("placement"), so might as well just make it an actual namespace, no? They are all static.

Reply:
Moved into namespace `placementFunctions` (to match map, match & opponent name style). Renamed them eg `placementCheckFF()` to `isAnyPlacementForfeit()`.